### PR TITLE
Change the name of locators in annotation FindBy.

### PR DESCRIPTION
**Pull request by Native1331**

### CODE REVIEW

**Comment by Native1331** [comment]
[RF | > Pages > Rename locators in annotation FindBy.](https://trello.com/c/TMHaPqEB/48-rf-pages-rename-locators-in-annotation-findby)

File: src/test/java/school/redrover/model/MultibranchPipelineConfigPage.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        return this;
+    }
+        public MultibranchPipelineConfigPage chooseBranchSources(){
+            addBranchSourcesMenu.click();
+        return this;
+    }
+
+
+
+
```
Comment (ruslanfursa) [inline src/test/java/school/redrover/model/MultibranchPipelineConfigPage.java:37]: Remove empty strings

File: src/test/java/school/redrover/model/MultibranchPipelineConfigPage.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    public MultibranchPipelineConfigPage displayNameOfMultibranchPipeline() {
+        displayNameField.sendKeys(nameAndDescriptionOfProject);
+        return this;
+    }
+    public MultibranchPipelineConfigPage description() {
+        descriptionTextarea.sendKeys(nameAndDescriptionOfProject);
+        return this;
+    }
+        public MultibranchPipelineConfigPage chooseBranchSources(){
```
Comment (ruslanfursa) [inline src/test/java/school/redrover/model/MultibranchPipelineConfigPage.java:30]: add space between ) {

**Review by ruslanfursa — APPROVED** [review]

File: src/test/java/school/redrover/model/MultibranchPipelineConfigPage.java (hunk 1 above)
Comment (Native1331) [reply src/test/java/school/redrover/model/MultibranchPipelineConfigPage.java:37]: Done!

File: src/test/java/school/redrover/model/MultibranchPipelineConfigPage.java (hunk 2 above)
Comment (Native1331) [reply src/test/java/school/redrover/model/MultibranchPipelineConfigPage.java]: Done!

**Pull request merged**